shared/src/vyxal/Elements.scala

```diff
@@ -3941,8 +3953,7 @@ object Elements:
           case n: VNum => n
           case other => throw BadArgumentException("reshape", other)
```

Comment:
This might possibly make the user think that `other` was the sole argument to reshape. Maybe we should have a more specific exception, like `s"bad argument to reshape, shape must be a number but got $shape"`? Not important, though, and also unrelated to this issue